src/qt/button.cpp

```diff
@@ -46,7 +46,14 @@ bool wxButton::Create(wxWindow *parent, wxWindowID id,
 {     
     QtCreate(parent);
     SetLabel( wxIsStockID( id ) ? wxGetStockLabel( id ) : label );
-
+    if( style & wxBU_LEFT )
+        m_qtPushButton->setStyleSheet( "Text-align:left" );
```

Comment:
Indentation here has one extra space, 9 in total. The same goes for the next 3 `else if` branches